packages/edit-site/src/components/post-edit/index.js

```diff
@@ -94,7 +94,12 @@ function PostEditForm( { postType, postId } ) {
 				'date',
 				'slug',
 				'parent',
-				'comment_status',
+				{
+					id: 'discussion',
+					label: __( 'Discussion' ),
+					children: [ 'comment_status', 'ping_status' ],
+					summary: 'discussion',
```

Comment:
I wonder if "summary" should be needed here at all, since it can just use the "id". cc @oandregal